docs/i2c.rst

```diff
@@ -21,6 +21,17 @@ should be described separately in that device's documentation.
 Functions
 =========
 
+.. py:function:: init(freq=None, sda=None, scl=None)
```

Comment:
This is different to spi.init: the spi always sets all parameters and has defaults for everything, so that calling spi.init() resets everything to the defaults.  I'd say i2c.init should follow that behaviour.